app/models/user.rb

```diff
@@ -470,6 +476,15 @@ def find_ids(options)
 
   protected
 
+  def promote_to_first_admin
+    account.update({first_admin_id: id})
+  end
+
+  def update_next_account_first_admin
+    return unless (another_admin_id = (account.admins.where.not(id: id) || []).first&.id)
+    account.update_column(:first_admin_id, another_admin_id) or raise ActiveRecord::ActiveRecordError
```

Comment:
Why are we skipping validations and callbacks here?

Reply:
Yes @guicassolato, although I am not sure if that's a good idea 🤔

Reply:
I am not sure if that's a good idea 🤔But it sounds good to me to raise an error if it fails, because otherwise it would be silently 🤔